source/Cosmos.System2/Graphics/SVGAIIScreen.cs

```diff
@@ -1,25 +1,33 @@
 using System;
 using System.Collections.Generic;
-using System.Text;
-using Cosmos.HAL.Drivers.PCI.Video;
 using System.Drawing;
 
+using Cosmos.Debug.Kernel;
+using Cosmos.HAL.Drivers.PCI.Video;
+
 namespace Cosmos.System.Graphics
 {
     public class SVGAIIScreen : Canvas
     {
+        private static readonly Mode DefaultMode = new Mode(1024, 768, ColorDepth.ColorDepth16);
+
+        private static readonly Debugger Debugger = new Debugger("System", "SVGAIIScreen");
+
+        private Mode _mode;
+
         public VMWareSVGAII xSVGAIIDriver;
-        internal Debug.Kernel.Debugger mSVGAIIDebugger = new Debug.Kernel.Debugger("System", "SVGAIIScreen");
 
-        public SVGAIIScreen() : base()
+        public SVGAIIScreen()
+            : this(DefaultMode)
         {
-            mSVGAIIDebugger.SendInternal($"Creting new SVGAIIScreen with default mode {defaultGraphicMode}");
+            Debugger.SendInternal($"Creting new SVGAIIScreen with default mode {DefaultGraphicMode}");
 
             xSVGAIIDriver = new VMWareSVGAII();
-            xSVGAIIDriver.SetMode((uint)defaultGraphicMode.Columns, (uint)defaultGraphicMode.Rows, (uint)defaultGraphicMode.ColorDepth);
+            xSVGAIIDriver.SetMode((uint)Mode.Columns, (uint)Mode.Rows, (uint)Mode.ColorDepth);
```

Comment:
If I'm not mistaking this(DefaultMode) already does:

xSVGAIIDriver = new VMWareSVGAII(); 
xSVGAIIDriver.SetMode((uint)Mode.Columns, (uint)Mode.Rows, (uint)Mode.ColorDepth);

in this case the ctor should have an empty body.